plugins/chat/assets/javascripts/discourse/services/chat-channel-pane.js

```diff
@@ -37,55 +40,19 @@ export default class ChatChannelPane extends Service {
     this.selectingMessages = true;
   }
 
-  @action
-  editMessage(newContent, uploads) {
```

Comment:
Ah I questioned why not have this editMessage code here in one of my other comments, what is the reasoning behind this?

Reply:
Yes I hesitated, in this case, the action is made on a message from the pane, but it really impacts the state of the composer so I thought It would make more sense to have it in the composer service. I expect we will be moving more things around in next weeks so it's possible it's a mistake here, not 100% sure of everything here yet.